packages/components/Notification/src/Notification.tsx

```diff
@@ -51,9 +49,10 @@ export const Notification = compose<NotificationType>({
     }, ['isBar']);
 
     return (final: NotificationProps, ...children: React.ReactNode[]) => {
-      const { variant, icon, title, action, ...rest } = mergeProps(userProps, final);
-      const iconProps = createIconProps(icon);
+      const { variant, icon, title, action, onActionPress, ...rest } = mergeProps(userProps, final);
       const mergedProps = mergeProps<PressableProps>(rest, rootStyle);
+      const iconProps = createIconProps(icon);
+      const notificationButtonProps = createNotificationButtonProps(userProps);
```

Comment:
Starting to sound awfully similar to a "useNotification" :D, we don't have to change that though. I'm still not sure what would count as "state" for us